docs/technical/application-structure.md

```diff
@@ -0,0 +1,131 @@
+# Application Structure
```

Comment:
**blocking**: i'm worried that people will miss that this isn't current, but future. Could we make this more obvious in the title? Something like "Aspirational Application Structure" or "Future Application Structure".

And now that I'm thinking about it, this isn't application structure so much as directory structure, so perhaps "Future Directory Structure" or "Future Repo Structure" makes more sense.

I'd like to see this title reflected in the filename too, for extra clarity.

Reply:
Updated in d22a096